tests/testsuite.py

```diff
@@ -200,6 +211,19 @@ def test_xmlrunner_buffer_output_fail(self):
         testsuite_output = self.stream.getvalue()
         self.assertIn('should be printed', testsuite_output)
 
+    def test_unittest_logger(self):
```

Comment:
if you wanted to test that some messages are logging, then you could install a handler temporarily that just puts the records in an array.